migrations/1597956943277-lernstore_view_permission.js

```diff
@@ -0,0 +1,46 @@
+const { connect, close } = require('../src/utils/database');
+const Roles = require('../src/services/role/model');
+
+module.exports = {
+	up: async function up() {
+		await connect();
+		await Roles.updateMany({
+			name: {
+				$in: [
+					'administrator',
+					'superhero',
+					'teacher',
+					'demo',
+					'demoTeacher',
+					'courseStudent',
+					'courseTeacher',
+					'courseSubstitutionTeacher',
+				]
+			},
+		}, {
+			$addToSet: { permissions: 'LERNSTORE_VIEW' },
+		});
+		await close();
+	},
+
+	down: async function down() {
+		await connect();
+		await Roles.updateMany({
+			name: {
+				$in: [
+					'administrator',
+					'superhero',
+					'teacher',
+					'demo',
+					'demoTeacher',
+					'courseStudent',
+					'courseTeacher',
+					'courseSubstitutionTeacher',
+				]
+			},
+		}, {
+			$pull: { permissions: 'LERNSTORE_VIEW' },
+		});
+		await close();
+	},
+};
```

Comment:
Codacy found an issue: [Newline required at end of file but not found.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6051638)